examples/README.md

```diff
@@ -14,17 +14,17 @@ include $(TOP)/mk/targets.mk
 
 * `NAME` - name of the example, e.g. `proxy`
 
-* `CONTAINERS` - list of all the containers to be built. Each of these should have a folder under `examples/<NAME>`, e.g. `examples/proxy/nginx`. This folder should have a file called `Dockerfile` to decribe how the container is built. The `Dockerfile` assumes to be executed from the top level project folder, i.e. the paths referenced in it should look like `examples/proxy/nginx/html`.
+* `CONTAINERS` - list of all the containers to be built. Each of these should have a folder under `examples/<NAME>`, e.g. `examples/proxy/nginx`. This folder should have a file called `Dockerfile` to describe how the container is built. The `Dockerfile` is assumed to be executed from the top level project folder, i.e. the paths referenced in it should look like `examples/proxy/nginx/html`.
 
 * `AUX_CONTAINERS` - specify a dependency container from another example. This allows for example to share container build definitions. The container name has to be in the form of `<DEP_NAME>-<DEP_CONTAINER>`, where `<DEP_NAME>` is the name of the dependency example, and `<DEP_CONTAINER>` is the name of the container within that example.
 
 * `PODS` - list of the pods to be deployed. Each of the names here should correspond to a file in the example's `k8s` folder. E.g. `examples/proxy/k8s/proxy-nsc.yaml`
 
-* `NETWORK_SERVICES` - a list of the Network Services to be deployed before the Endpoints that implement them. Each of the names here should correspond to a file in the example's `k8s` folder. E.g. `examples/secure-intranet/k8s/secure-intranet.yaml`
+* `NETWORK_SERVICES` - a list of the Network Services to be deployed before the Endpoints that implement them. Each of the names here should correspond to a file in the example's `k8s` folder. E.g. `examples/secure-intranet/k8s/secure-intranet.yaml`. The Network Services can also be implicitly created when endpoint annotations indicate the endpoint implements the service.
```

Comment:
Hmm I get what you say here and why. Still I'm not sure this is the right explanation. Which annotations do you refer to?